File: v2/nodejs/video/main.js
const utils = require('../lib/utils')

const url = 'http://api-video-bj.fengkongcloud.com/v2/saas/anti_fraud/video'
const access_key = '{ACCESS_KEY}'
const bt_id = '{BT_ID}'
const video_url = 'https://jsonplaceholder.typicode.com/posts/'

const payload = {
    'accessKey': access_key,
    'imgType': 'POLITICS_PORN_AD',
    'audioType': 'NONE',
    'appId': 'default',
    'btId': bt_id,
    'callback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        'url': video_url,
    },
}

utils.sendRequest(url, 'POST', payload, data => {
    // 通过 code 和 riskLevel 判断返回，具体请参考接口文档
    console.log(data)
})

File: v2/lua/video/main.lua
local json = require('cjson')
local hc = require('lib.httpclient').new()

local url = 'http://api-video-bj.fengkongcloud.com/v2/saas/anti_fraud/video'
local accessKey = '{ACCESS_KEY}'

local payload = json.encode({
  accessKey = accessKey,
  imgType = 'POLITICS_PORN_AD',
  audioType = 'NONE',
  appId = 'default',
  btId = '{BT_ID}',
  callback = 'https://jsonplaceholder.typicode.com/posts/',
  data = {
    url = 'https://jsonplaceholder.typicode.com/posts/',
  }
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])
